sdk/eventgrid/Azure.Messaging.EventGrid/src/autorest.md

```diff
@@ -49,4 +49,5 @@ input-file:
     -  https://github.com/Azure/azure-rest-api-specs/blob/00ac1cbffba123ba5e30cb324935100495d0700d/specification/eventgrid/data-plane/Microsoft.MachineLearningServices/stable/2018-01-01/MachineLearningServices.json
     -  https://github.com/Azure/azure-rest-api-specs/blob/00ac1cbffba123ba5e30cb324935100495d0700d/specification/eventgrid/data-plane/Microsoft.Cache/stable/2018-01-01/RedisCache.json
     -  https://github.com/Azure/azure-rest-api-specs/blob/00ac1cbffba123ba5e30cb324935100495d0700d/specification/eventgrid/data-plane/Microsoft.Web/stable/2018-01-01/Web.json
+    -  https://github.com/Azure/azure-rest-api-specs/blob/d587005c88aaa684cd744fe32019ad5f8f51dd98/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json
```

Comment:
Any reason you don't reference the readme directly https://github.com/Azure/azure-rest-api-specs/blob/d587005c88aaa684cd744fe32019ad5f8f51dd98/specification/eventgrid/data-plane/readme.md?

Reply:
I think this may have been a holdover from when we were using Matt's unofficial swagger.

Reply:
Actually, we are still using Matt's swagger. For some reason, I thought this had been merged in.